### PR TITLE
Split abis

### LINKED ISSUE #53: Split BridgeFungibleToken abi into two parts
Currently we have this:
`impl BridgeFungibleToken for Contract {`

We should extract the bridge-specific stuff out into `abi Bridge {`
and then implement the WIP `abi FRC20 {` so the contract adheres to the new(coming soon TM) standard.

### PR DESCRIPTION
**Pull request by nfurfaro**
These refactors set us up to use seperate abis for `Bridge` and `Fungible Token` functionality.
The FRC20 abi is currently a local lib, but we can fetch this dep externally when it becomes available without touching the contract itself (ContractId may still change though).
closes #53

### CODE REVIEW

File: bridge-fungible-token/tests/utils/environment.rs [hunk 1]
```diff
@@ -15,53 +15,53 @@ use fuels::{
     prelude::*,
     types::{message::Message, Bits256},
 };
-use primitive_types::U256;
+use primitive_types::U256 as Unsigned256;
```
Comment (pixelcircuits) [inline bridge-fungible-token/tests/utils/environment.rs:18]: Why did you add an alias?

File: bridge-abi/Forc.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [project]
 authors = ["Fuel Labs <contact@fuel.sh>"]
-entry = "bridge_fungible_token_abi.sw"
+entry = "bridge_abi.sw"
```
Comment (pixelcircuits) [inline bridge-abi/Forc.toml:3]: I kinda feel like `bridge_abi` is too generic because there will eventually be a non-fungible bride as well that will need a different abi

File: bridge-fungible-token/tests/utils/environment.rs (hunk 1 above)
Comment (nfurfaro) [reply bridge-fungible-token/tests/utils/environment.rs:18]: The Rust `U256` was conflicting with the Sway `U256` name found in the Contract module.
